hack/start-agent.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+usage()
+{
+cat << EOF
+Usage: $0 OPTIONS
+This script runs test for flotta-agent to verify it can start running properly.
+OPTIONS:
+   -d      Path to flotta-device-worker directory (optional)
+   -h      Show this message
+   -t      Target (ocp/k8s)
+   -y      Path to yggdrasil directory
+EOF
+}
+
+while getopts "t:y:d:" option; do
+    case "${option}"
+    in
+        t) TARGET=${OPTARG};;
```

Comment:
TBH: Agents shouldn't know if TARGET is ocp or k8s at all. At the end is only an API.